### ISSUE TITLE
Compiling src_module_network

### ISSUE THREAD

**Issue by kxzaon** [Alexays/Waybar#151]
```
ninja: Entering directory `build'
[42/44] Compiling C++ object 'waybar@exe/src_modules_network.cpp.o'.
FAILED: waybar@exe/src_modules_network.cpp.o 
c++ -Iwaybar@exe -I. -I.. -I../include -I../subprojects/fmt-5.2.1/include -I/usr/include/libdrm -I/usr/include/pixman-1 -I/usr/include/sigc++-2.0 -I/usr/lib/sigc++-2.0/include -I/usr/include/gtkmm-3.0 -I/usr/lib/gtkmm-3.0/include -I/usr/include/atkmm-1.6 -I/usr/include/atk-1.0 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/glibmm-2.4 -I/usr/lib/glibmm-2.4/include -I/usr/include/giomm-2.4 -I/usr/lib/giomm-2.4/include -I/usr/include/pangomm-1.4 -I/usr/lib/pangomm-1.4/include -I/usr/include/cairomm-1.0 -I/usr/lib/cairomm-1.0/include -I/usr/include/cairo -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/uuid -I/usr/include/pango-1.0 -I/usr/include/fribidi -I/usr/include/harfbuzz -I/usr/include/gtk-3.0 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/gio-unix-2.0/ -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib/dbus-1.0/include -I/usr/include/gtk-3.0/unix-print -I/usr/include/gdkmm-3.0 -I/usr/lib/gdkmm-3.0/include -I/usr/include/libnl3 -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -std=c++17 -O3 '-DVERSION="0.2.3-36-gaedf133 (" __DATE__ ", branch '"'"'master'"'"')"' -DHAVE_SWAY -DHAVE_LIBNL -pthread  -MD -MQ 'waybar@exe/src_modules_network.cpp.o' -MF 'waybar@exe/src_modules_network.cpp.o.d' -o 'waybar@exe/src_modules_network.cpp.o' -c ../src/modules/network.cpp
../src/modules/network.cpp: In member function 'int waybar::modules::Network::netlinkRequest(void*, uint32_t, uint32_t)':
../src/modules/network.cpp:350:65: error: cannot convert 'std::nullptr_t' to 'int' in initialization
   struct msghdr msg = { &sa, sizeof(sa), &iov, 1, nullptr, 0, 0 };
                                                                 ^
../src/modules/network.cpp: In member function 'int waybar::modules::Network::netlinkResponse(void*, uint32_t, uint32_t)':
../src/modules/network.cpp:361:65: error: cannot convert 'std::nullptr_t' to 'int' in initialization
   struct msghdr msg = { &sa, sizeof(sa), &iov, 1, nullptr, 0, 0 };
                                                                 ^
[43/44] Compiling C++ object 'waybar@exe/src_client.cpp.o'.
ninja: build stopped: subcommand failed.

```
OS : Alpine x86_64
I'm using the package libnl3-dev for the network. I'm not an expert of compiling so i'm confused about this error.

Thanks.

**Comment by kxzaon** [comment]
I figured it out. It seems to be related to musl. https://www.openwall.com/lists/musl/2013/01/09/1
I had to add "#define nullptr 0" in order to work.
I don't think it is a proper way, I'm not a developer, but it works.

**Comment by Alexays** [comment]
Fixed with e12766a656731b9582ac3020b511db0ad5aced8f

**Comment by kxzaon** [comment]
I confirm. Thanks.
